handbook/data-stores/data-stores.md

```diff
@@ -0,0 +1,115 @@
+1. Click **Test Connection** to ensure SystemLink can connect to the MongoDB instance.
+1. If the connection test was successful, click **Apply** to restart SystemLink service manager and connect to the MongoDB instance.
+
+!!!important
+    Some connection string formats are unsupported or could cause issues in some environments.
+
+    - Connection strings that contain the query parameter `tls=` are unsupported. Use the `ssl=` query string to achieve the same capability with the same degree of security.
+    - The DNS seed list URI format (`mongodb+srv://`) may fail in some environments due to a bug ([Unable to connect to Atlas due to DNS connectivity issues #358](https://github.com/kobil-systems/mongodb/issues/358)) in the Elixir MongoDB driver used by SystemLink. In this case use the [MongoDB standard connection string format](https://docs.mongodb.com/manual/reference/connection-string/#std-label-connections-standard-connection-string-format) to connect your replica set. This affects both self-hosted replica sets and MongoDB Atlas.
+    - When URL escaping characters in your connection string  you must use uppercase characters, e.g. `%2F` not `%2f`.
+
+    Example connection string (line breaks for readability):
+    
+    ```url
+    mongodb://myusername:<password>:
+    ec2-123-123-12.compute-1.amazonaws.com27017,
+    ec2-234-234-23.compute-1.amazonaws.com:27017,
+    ec2-456-456-45.compute-1.amazonaws.com:27017/
+    ?authSource=admin
+    &replicaSet=rs0
+    &readPreference=primary
+    &ssl=true
+    ```
+
+### Connecting to MongoDB Atlas
+
+MongoDB Atlas is a fully managed PaaS from MongoDB. Atlas provides for simpler setup and adminstration of a MongoDB replica set than self hosting.
+
+If you have not setup an Atlas cluster before, refer to [Getting started with Atlas](https://docs.atlas.mongodb.com/getting-started/). Refer to [Setup Atlas Connectivity](https://docs.mongodb.com/guides/cloud/connectionstring/) for steps to obtain a connection string.
+
+!!!note "MongoDB Atlas free tier is unsupported by SystemLink"
+    Due to constraints on memory in the Atlas free tier, SystemLink will not successfully connect to a cluster running on the free tier. You must use a paid tier to successfully connect. The lowest performance paid tiers of Atlas are supported by SystemLink. If you need assistance evaluating Atlas with SystemLink please contact NI ([customer.requests@ni.com](mailto:customer.requests@ni.com)) or your local account manager.
+
+Because Atlas uses replica sets by default, your SystemLink server could be affected by the bug in the Elixir MongoDB driver used by SystemLink as described in [Connecting to MongoDB with Replica Sets](#connecting-to-mongodb-with-replica-sets). When obtaining a connection string from Atlas use the connection string generated for the Node.js driver version 2.0.14. This provided connection string is fully supported by SystemLink.
```

Comment:
The other connection string gotchas also apply to Atlas. Would it be worth pulling them into a separate section that can be linked from both places?